fetch.bs

```diff
@@ -5630,6 +5624,71 @@ if the script checks that the URL has the right hostname.
 
 
 
+<h2 id=data-urls><code>data:</code> URLs</h2>
+
+<p>For an informative description of <code>data:</code> URLs, see RFC 2397. This section replaces
+that RFC's normative processing requirements. [[RFC2397]]
```

Comment:
Some notes on why this is necessary can be helpful. Citing @SimonSapin's document on RFC 2397 ambiguities as an informative reference should suffice.

Reply:
That document will go away. But yeah, we should include some basic rationale.

Reply:
Done.